src/main.ts

```diff
@@ -90,6 +70,26 @@ export default class BreadcrumbsPlugin extends Plugin {
 		this.app.workspace.onLayoutReady(async () => {
 			log.debug("on:layout-ready");
 
+			// Set the edge_fields & BC-meta-fields to the right Properties type
+			try {
+				const all_properties =
+					this.app.metadataTypeManager.getAllProperties();
+	
+				for (const field of this.settings.edge_fields) {
+					if (all_properties[field.label]?.type) continue;
```

Comment:
`type` will always be there, so this if branch will always `continue`. I understand what you intend to do, but don't think this achieves it. 
Maybe try something like `if (!["text", "multitext"].includes(all_properties[field.label]?.type)) { continue }`. In other words, if the field type has already been set to something other than "text" (the default), or "multitext" (the BC target), then skip it. The assumption being that the user overrode that field

Reply:
Won't type only be there if the property has already been set? Meaning that if the `properties.yaml` doesn't contain the property field yet, a type won't be set, and it will then run `this.app.metadataTypeManager.setType(field.label, "multitext");`.

However the only two types that make sense here are text or multitext, so I'd be fine with changing this to your suggestion, I can modify it.